### PR TITLE
Updates for 1.40 readme and changelog

### PR DESCRIPTION
**Pull request by erpett**
Readme and Changelog updates for 1.40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +16,19 @@ Azure Data Studio is a data management tool that enables you to work with SQL Se
...
 |Mac      |.zip             |[Intel Chip][osx-zip] |
-
-[win-user]: https://go.microsoft.com/fwlink/?linkid=2204567
-[win-system]: https://go.microsoft.com/fwlink/?linkid=2204568
-[win-zip]: https://go.microsoft.com/fwlink/?linkid=2204772
-[osx-zip]: https://go.microsoft.com/fwlink/?linkid=2204569
-[linux-zip]: https://go.microsoft.com/fwlink/?linkid=2204773
-[linux-rpm]: https://go.microsoft.com/fwlink/?linkid=2204774
-[linux-deb]: https://go.microsoft.com/fwlink/?linkid=2204570
+|Mac      |.zip             |[Apple Silicon][osx-arm64] |
```
Comment (alanrenmsft) [inline README.md:21]: please follow the style in the insiders section, there should be just one row for Mac.
image

Comment (alanrenmsft) [reply README.md:21]: `&emsp;` here means four spaces

Comment (alanrenmsft) [reply README.md:21]: If you look closely you will find the other platforms do have different type of packages, and Mac has just one `.zip`, this is similar to how VSCode organize their downloads.

image

Comment (alanrenmsft) [reply README.md]: I've made the change directly in your branch.

**Review by alanrenmsft — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -16,15 +16,17 @@ Azure Data Studio is a data management tool that enables you to work with SQL Se
...
-|Mac      |.zip             |[Intel Chip][osx-zip] |
-
-[win-user]: https://go.microsoft.com/fwlink/?linkid=2204567
-[win-system]: https://go.microsoft.com/fwlink/?linkid=2204568
-[win-zip]: https://go.microsoft.com/fwlink/?linkid=2204772
-[osx-zip]: https://go.microsoft.com/fwlink/?linkid=2204569
-[linux-zip]: https://go.microsoft.com/fwlink/?linkid=2204773
-[linux-rpm]: https://go.microsoft.com/fwlink/?linkid=2204774
-[linux-deb]: https://go.microsoft.com/fwlink/?linkid=2204570
+|Mac      |.zip             |[Universal][osx-universal]&emsp;[Intel Chip][osx-zip]&emsp;[Apple Silicon][osx-arm64]|
```
Comment (kburtram) [inline README.md:19]: not sure this is correct format with `&emsp;` tags.  Probably could us a similar format to other entries, i.e. on download per row.

**Review by kburtram — APPROVED** [review]
A couple comments about typo and link formatting in README file.

**Pull request merged**